.editorconfig

```diff
@@ -0,0 +1,25 @@
+[*]
+charset = utf-8
+end_of_line = lf
+ij_continuation_indent_size = 8
```

Comment:
should this line be under `[*.java]`  block?

Reply:
moving this to java block should work. If a further adjustment is needed we gonna find out.
Thanks.